es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/BulkRequestBuilderWrapper.java

```diff
@@ -30,7 +31,7 @@
 public class BulkRequestBuilderWrapper {
 	private final BulkRequestBuilder bulkRequestBuilder;
```

Comment:
We cannot use atomic in this case because it is underlying ArrayList and it variables that need be atomic so that different threads read the most recent write.  Please correct me if I'm wrong.